contrib/locale/build_cleanlocale.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+CONTRIB_LOCALE="$(dirname "$(readlink -e "$0")")"
```

Comment:
The version of readlink installed on macOS does not support the -e flag, so there's a lot of noise when running argbash commands:

```
💬 INFO:  preparing electrum-locale.
readlink: illegal option -- e
usage: readlink [-fn] [file ...]
```

On a mac you can install a version of readlink that argbash expects: Using homebrew, `brew install coreutils` will install `greadlink` which supports the `-e` flag.

Reply:
@SomberNight ^

Reply:
Thanks.
See https://github.com/spesmilo/electrum/commit/3a5815d8548b7fe0318de4f7c489a896bebe2f61